ballerina/introspection_validator_visitor.bal

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import graphql.parser;
+
+class IntrospectionValidatorVisitor {
```

Comment:
I think we should move all the introspection-related validations inside this visitor. It will make the code more readable and may reduce the load of the field validator visitor. We can do it in a separate PR though. Shall we create an issue to track this?

Reply:
Tracked via [ballerina-standard-library#3348](https://github.com/ballerina-platform/ballerina-standard-library/issues/3348)